packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2665,6 +2666,77 @@ class SizedOverflowBox extends SingleChildRenderObjectWidget {
 /// needed, prefer removing the widget from the tree entirely rather than
 /// keeping it alive in an [Offstage] subtree.
 ///
+/// {@tool dartpad --template=stateful_widget_material_ticker}
+///
+/// This example shows a [FlutterLogo] widget when the `_offstage` member field
+/// is false, and hides it without any room in the parent when it is true. The
+/// [Size] of the logo without bringing it on screen will be measured.
+///
+/// ```dart
+/// class OffstageLogo extends StatefulWidget {
+///   @override
+///   _OffstageLogoState createState() => _OffstageLogoState();
+/// }
+///
+/// class _OffstageLogoState extends State<OffstageLogo> {
```

Comment:
This section isn't necessary (lines 2676 to 2681) because the template you're using already assumes a stateful widget.

See the logs here for details: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8868703876011722080/+/steps/prepare_environment/0/steps/Analyze/0/steps/analyze/0/stdout